docs/manual/annotating-libraries.tex

```diff
@@ -844,6 +844,69 @@
 in the version of the library you are using.
 
 
+\sectionAndLabel{Ajava Files}{ajava-files}
```

Comment:
Should  https://checkerframework.org/manual/#whole-program-inference be updated to reference the new file format?

Reply:
Currently, the `wpi.sh` and `wpi-many.sh` scripts only support stub files. It should be easy to update them to use ajava files, but that can be done separately.